docs/src/install.md

```diff
@@ -6,6 +6,32 @@ to install the latest stable version of this package:
 ```julia
 pkg> add Bloqade
 ```
+## Using Bloqade Optimized AMIs from AWS Marketplace
+The Bloqade team has implemented 2 dedicated AMIs that can be acquired using AWS Marketplace
+
+- Bloqade AMI with Julia
+- Bloqade Optimized Deep Learning AMI with CUDA and Julia
+
+
+### Bloqade AMI with Julia (base image)
+The Bloqade AMI contains:
+- The latest Julia installation using juliaup '1.5.1' which Julia '1.5.1'  for more information please refer to [Juliaup](https://github.com/JuliaLang/juliaup) 
+- The latest version of Bloqade 
+
+### Bloqade Optimized Deep Learning AMI with CUDA and Julia
+In addition to the content of our base image, this image further contains
+- NVIDIA CUDA, cuDNN, NCCL, GPU Drivers, Intel MKL-DNN, Docker, NVIDIA-Docker and EFA support
+- Block devices
+ /dev/sda1=snap-03d72fbeb983a4663:60:true:gp2
+ /dev/sdb=ephemeral0
+ /dev/sdc=ephemeral1
+
+
+### Bloqade AMI
+To use the AMI 
```

Comment:
I think a warning block would be better to notify the users that the AMI is not public ready yet?

Reply:
yes agreed let me tweek it that was my concern we change it once it is official